### ISSUE TITLE
[Quote]: QA: Two way scroll bar is observed at 400% zoom

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8971]
### Description

Page Element:
Magnifier button

Issue Description:
When the page is zoomed to 400%, two way scroll bar is observed in the page

Expected:
When the page is zoomed to 400%, only vertical scroll bar should be observed.

User Impact:
When two way scroll bar is observed in the application, user with low vision who uses zoom feature will face difficulty in accessing the application

### Component(s) impacted

Quote

### Browser

Firefox

### Carbon for IBM.com version

v1.34.0-rc.0

### Severity

Severity 1 = The design is broken in a critical way that blocks users from completing tasks or damages the brand. Affects major functionality, no workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-react

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components-react

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
